File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        // if (nums.empty())
        //     return 0;
        // int k=1;
        // for(int i=1;i<nums.size();i++)
        // {
        //     if (nums[i]!=nums[i-1])
        //         nums[k++]=nums[i];
        // }
        // return k;
        
        int i=0;
        for(int num:nums){
            if(i==0 or nums[i-1]!=num)
                nums[i++]=num;
        }
        return i;
    }
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        i=0
        for n in range(len(nums)):
            if n==0 or nums[n]!=nums[i-1]:
                nums[i]=nums[n]
                i+=1
        return i
        